pyartcd/pyartcd/locks.py

```diff
@@ -143,6 +148,36 @@ async def get_locks(self, version: str = None):
         return await redis.get_keys(pattern)
 
 
+async def enqueue_for_lock(coro: coroutine, lock: Lock, lock_name: str, lock_id: str,
```

Comment:
Is there a reason for us to have this function that only serves as a wrapper to `_enqueue_for_lock`? Can't this be called directly?

Reply:
This is to cache lock manager instance. I wanted to cache redis connection as well, but that will take some refactoring